scheduler/scripts/run_main.py

```diff
@@ -69,12 +70,18 @@ def main(*,
         night_events = collector.get_night_events(site)
         for night_idx in night_indices:
             eve_twilight_time = night_events.twilight_evening_12[night_idx].to_datetime()
-            eve_twilight = EveningTwilight(start=eve_twilight_time, reason='Evening 12° Twilight', site=site)
+            eve_twilight = EveningTwilight(event_id=uuid.uuid4(),
```

Comment:
If we auto-generate the `uuid` like above, then we don't have to keep initializing this field.